docs/concepts/pools/managed.md

```diff
@@ -203,4 +203,31 @@ _managedPool.setJoinExitEnabled(true);
 
 // Disable joins and exits
 _managedPool.setJoinExitEnabled(false);
+```
+
+## Circuit Breakers
+Circuit breakers are risk management infrastructure that can be used to protect pools in the event that a token becomes compromised and the value of the token changes drastically, with respect to the other tokens in the pool. Managed Pool `owner`s can set a lower bound and an upper bound for each token, which are percentages corresponding to a relative change in the token's spot price. If the token's price change falls outside of the bounds, the circuit breaker will trip and prevent swaps that would result in the value of the token dropping below the lower bound or rising above the upper bound. All token prices are in terms of BPT price for the given token. This ensures prices are calculated relative to the rest of the tokens in a pool. For example, if a circuit breaker is set with a lower bound of 0.7 and an upper bound of 2, and the token's BPT price drops by 30% or doubles in value, the circuit breaker will be "tripped", and the transaction will be reverted. For more information on how BPT prices are calculated, see [valuing-bpt](../advanced/valuing-bpt/valuing-bpt.md). For stable coins and assets that are closely correlated in price is recommended to have a very narrow range for the circuit breaker bounds. For assets that are less correlated, it is recommended for the `owner` to set the bounds to a wider range.
```

Comment:
```suggestion
Circuit breakers are risk management infrastructure that can be used to protect pools in the event that the value of a token changes drastically with respect to the other tokens in the pool. Managed Pools have upper and lower bounds for each token, and the `owner` can set one, both, or neither bound on a per-token basis. The bounds are percentages corresponding to a relative change in the BPT price with respect to the token. If a given operation (swap, join, exit) will result in a token's price changing to a value outside of the bounds, the circuit breaker will prevent that operation from happening. Since BPT prices are with respect to tokens, prices are ultimately calculated relative to the rest of the tokens in a pool. For example, consider a situation in which a token's circuit breaker is set with a lower bound of `0.7` and an upper bound of `2.0`. The pool will allow operations to continue as the token-denominated BPT price approaches a 30% value decrease or 2x value increase, but if a user attempts an operation that would cause the price to cross either threshold, the circuit breaker will prevent that from happening, and the transaction will be reverted. 

To get BPT price with respect to a single token, users can calculate this as
$$ \frac{supply_{BPT} * weight_{token}}{balance_{token}} $$
::: warning Don't use this calculation for any other BPT pricing technique!
This calculation is strictly for circuit breaker bounds. 
:::

For stablecoins and assets that are closely correlated in price, `owner`s can set a relatively narrow range for the circuit breaker bounds. For assets that are less correlated, `owner` can decide how they want to balance token volatility and range enforcement.
```